src/ui/game_window.c

```diff
@@ -95,9 +96,10 @@ void handleGameEvents()
     handlePacmanEvents();
 }
 
-void startReady()
+void startReady(int initialCount)
 {
     isGamePause = true;
+    readyTimer.initialCount = initialCount;
```

Comment:
this is illegal !!!! but ok 
![help](https://github.com/Livl-Corporation/livl-pacman/assets/19238963/aa497df3-1628-4d56-86f7-a5eccc24dab8)

we can maybe create a function in timer.h to change timer duration

Reply:
HAHHAHAHA YEAH, very illegal but yep I will create a function for timer duration change.

Reply:
![image](https://github.com/Livl-Corporation/livl-pacman/assets/62793491/92eb53ca-afd6-4b7b-b451-62660c089d46)
if this is illegal, then doing xxTimer->callback = yyy is also illegal so I created a setter function as well.

Reply:
fe4084c